index.html

```diff
@@ -220,7 +220,53 @@ <h3>Skills</h3>
     </main>
 
     <footer>
+        <h3 class="footer_title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        <form action="https://formspree.io/f/xgebnwyd" method="POST" class="flex_column">
+            <ul class="flex_column">
+                <li class="form_list">
+                    <input 
+                    type="text" 
+                    id="name" 
+                    name="user_name" 
+                    placeholder="Full name" 
+                    minlength="2" 
+                    maxlength="30" 
+                    required>
+                </li>
+                <li class="form_list">
+                    <input 
+                    type="email" 
+                    name="user_email" 
+                    id="email" 
+                    placeholder="Email address" 
+                    required>
+                </li>
+                <li>
+                    <textarea 
+                    name="user_message" 
+                    id="message" 
+                    placeholder="write something..." 
+                    minlength="2" 
+                    maxlength="" 
+                    required></textarea>
+                </li>
+                <li class="form_button flex">
+                    <button type="submit" class="button_enabled">Get in touch</button>
```

Comment:
- [ ] Nice one applying the `hover` and `enabled` effects to the button, but it seems the `clicked` effect which is a bit darker than the `enabled` is not been applied, or is it my eye that is deceiving me 🤣.

Kindly fix this, and likewise the `disabled`, to ensure a pixel-perfect design, you can use the [developer's best friend](https://www.google.com/) to make your research.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/228265780-ea1d4a41-c153-4eac-93d4-bed34434cfc5.gif)

- Figma design
<img width="274" alt="button2" src="https://user-images.githubusercontent.com/41163571/227492511-ac99583d-add1-41a8-b99f-2b2f12771941.png">